File: src/DemoWhatsAppNETAPICSharp/FrmListContact.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;

namespace DemoWhatsAppNETAPICSharp
{
    public partial class FrmListContact : Form
    {
        public FrmListContact(IList<string> contacts)
        {
            InitializeComponent();

            var noUrut = 1;

            foreach (var contact in contacts)
            {
                lstContact.Items.Add(string.Format("{0}. {1}", noUrut, contact));
                noUrut++;
            }
        }

        private void btnClose_Click(object sender, EventArgs e)
        {
            this.Close();
        }
    }
}

File: src/DemoWhatsAppNETAPIVBNET/FrmListContact.vb
Public Class FrmListContact

    Public Sub New(ByVal contacts As IList(Of String))
        ' This call is required by the designer.
        InitializeComponent()

        Dim noUrut = 1

        For Each contact As String In contacts

            lstContact.Items.Add(String.Format("{0}. {1}", noUrut, contact))
            noUrut += 1
        Next

    End Sub

    Private Sub btnClose_Click(sender As Object, e As EventArgs) Handles btnClose.Click
        Me.Close()
    End Sub
End Class